README.md

```diff
@@ -20,9 +20,39 @@ m2m is a python package and command line tool to assist in converting csv docume
 </metadata>
 ```
 
-# License
+Requiremnets
+------------
 
-See LICENSE.txt.
+* Python 3.6 - 3.8
+
+Installation
+--------------
+
+This application can be installed by following the steps below:
```

Comment:
Maybe "can be installed and run" since you have a command to run it as well as install.